expressEngine/model/collection/order.js

```diff
@@ -25,6 +29,12 @@ const ordersSchema = new Schema({
   collection: 'orders',
   timestamp: true,
 });
+
+ordersSchema.pre('save', async function preSave() {
```

Comment:
If you're not using await then why you use async.

Reply:
update it